kbs/docker/lighthouse/assets/start-lighthouse.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# TODO: If desired, this can be beefed up (take the kbs port as an arg, check
+# the args themselves, etc.)
+
+kbs_url=$1
+lighthouse_vpn_ip=$2
+
+kbs_port=8080
+lighthouse_prefix_len=24
+
+# get creds for the lighthouse from the kbs
+/usr/local/bin/kbs-client \
+  --url http://${kbs_url}:${kbs_port} \
+  get-resource \
+  --path "plugin/nebula-ca/credential?name=nebula-lighthouse&ip=${lighthouse_vpn_ip}/${lighthouse_prefix_len}" \
```

Comment:
This may require plugin support in the kbs-client.
See the commit message of https://github.com/confidential-containers/trustee/pull/539, section `How to test the credential service`

Reply:
You're right. Will address this in a separate PR.